docs/sources/writing-guide/contribute-documentation/index.md

```diff
@@ -54,21 +54,45 @@ Small changes might include:
 
 > **Note:** You can only edit a topic that is part of the latest release of documentation.
 
-## Before you begin
+### Before you begin
 
 - Create a [GitHub](https://www.github.com) account.
 
 **To edit a topic:**
 
 1. From a topic on the documentation website, click **Edit this page** (pencil icon).
-2. Enter your changes.
-3. Change the branch name, if required.
+1. Enter your changes.
+1. Change the branch name, if required.
    The branch name is auto-populated.
 
-4. Click **Propose changes**.
+1. Click **Propose changes**.
 
    A PR is created and then goes through the PR review and approval workflow.
 
+## Develop a new topic
+
+If you want to develop a new topic from scratch, you can create a documentation plan and collaborate with a member of the Docs team. According to the book [_Docs for Developers_](https://docsfordevelopers.com/), a documentation plan is a _flexible outline_ for anticipating where the writing process will lead you.
+
+Your documentation plan helps you to:
+
+- Identify existing information gaps, and explain how you will fill them.
+- Get feedback from users and stakeholders before the writing process begins.
+- Consider different approaches you might take, and decide on one of them.
+
+> **Note:** Only create new topics that are part of the upcoming release of documentation.
+
+**To develop a new topic:**
+
+1. Create a new document in Google Docs, Google Sheets, or a similar tool. For a small project, you may use an email message.
+1. Copy and paste content from the [Document Planning Template]({{< relref "../../static/templates/document-planning-template.md) into your document. 
+1. Fill in the requested information. 
+1. Attach an outline of your proposed document. 
+1. Schedule a review meeting with a member of the Docs team.
+
+   If you don't know which writer will be reviewing your documentation plan, reach out to [docs@grafana.com](mailto:docs@grafana.com). 
```

Comment:
Fiona is putting boundaries around what the team can and cannot do with our current staffing, but this makes it sound like we have the bandwidth to collaborate on any docs requests that come in.

Reply:
This is definitely a valid concern. One way to resolve it would be to delete the content about "developing a new topic" entirely. Another way would be to add caveats to the effect that all requests will be triaged.